lib/segment/src/segment/search.rs

```diff
@@ -91,7 +91,7 @@ impl Segment {
     #[cfg(feature = "testing")]
     pub fn search(
         &self,
-        vector_name: &str,
+        vector_name: &crate::types::VectorName,
```

Comment:
Used the qualified path here on purpose?

Reply:
Moved to the top of the file for consistency. (it was here because it's behind `#[cfg]`)